test/e2e/network/ingress.go

```diff
@@ -153,6 +154,29 @@ var _ = SIGDescribe("Loadbalancing: L7", func() {
 			// framework.ExpectNoError(jig.verifyURL(fmt.Sprintf("https://%v/", ip), "", 30, 1*time.Second, httpClient))
 		})
 
+		It("multicluster ingress should get instance group annotation", func() {
+			name := "echomap"
+			jig.CreateIngress(filepath.Join(framework.IngressManifestPath, "http"), ns, map[string]string{
+				"kubernetes.io/ingress.class": "gce-multi-cluster",
+			}, map[string]string{})
+
+			By(fmt.Sprintf("waiting for Ingress %s to come up", name))
+			pollErr := wait.Poll(2*time.Second, framework.IngressReqTimeout, func() (bool, error) {
+				ing, err := f.ClientSet.ExtensionsV1beta1().Ingresses(ns).Get(name, metav1.GetOptions{})
+				framework.ExpectNoError(err)
+				annotations := ing.Annotations
+				if annotations == nil || annotations[instanceGroupAnnotation] == "" {
+					framework.Logf("Waiting for ingress to get %s annotation. Found annotations: %v", instanceGroupAnnotation, annotations)
```

Comment:
Can we make this work with NEG too? Is NEG going to put a NEGAnnotation instead of an instanceGroupAnnotation?

@freehan

Reply:
Currently there is no logic to put NEGAnnotation in ingress controller.

@nikhiljindal Can you pick this up? I will be out starting next week.

Reply:
Yes we can make it work with NEGs and if we do then yes we will get back a NEGAnnotation instead of instance group annotation.
As @freehan said we need to add that logic.